include/ppx/knob.h

```diff
@@ -54,16 +54,15 @@ class Knob
     const std::string& GetFlagName() const { return mFlagName; }
     const std::string& GetFlagHelp() const { return mFlagDesc; }
     size_t             GetIndent() const { return mIndent; }
+    const std::string& GetHelpParams() const { return mHelpParams; }
 
     void SetDisplayName(const std::string& displayName) { mDisplayName = displayName; }
     void SetFlagDesc(const std::string& flagDesc) { mFlagDesc = flagDesc; }
     void SetIndent(size_t indent) { mIndent = indent; }
+    void SetHelpParams(const std::string& helpParams) { mHelpParams = helpParams; }
```

Comment:
Looking at just changes here and not how this will be used later, how do you feel about overriding `GetHelpParams` to build the string instead of having setter/getter. Perhaps later usage of SetHelpParams changes things though